python/pfs/utils/fiberids.py

```diff
@@ -91,7 +92,7 @@ def load(self, path):
                  ('rad', 'f4'),
                  ('spectrographId', 'u2'),
                  ('fiberHoleId', 'u2'),
-                 ('scienceFiberId', 'u2'),
+                 ('scienceFiberId', 'U4'), # we change this to int after parsing the strings
```

Comment:
Need two spaces before a comment to satisfy linter. (Maybe doesn't matter for pfs_utils.)